src/main/java/com/google/sps/servlets/ChatServlet.java

```diff
@@ -14,9 +14,11 @@
 @WebServlet("/chat")
 public final class ChatServlet extends HttpServlet {
 
-  final private String APP_ID = "1024158";
-  final private String CLIENT_KEY = "d15fbbe1c77552dc5097";
-  final private String CLIENT_SECRET = "91fd789bf568ec43d2ee";
+  final static private String APP_ID = "1024158";
+  final static private String CLIENT_KEY = "d15fbbe1c77552dc5097";
+  final static private String CLIENT_SECRET = "91fd789bf568ec43d2ee";
+  final static private String PUSHER_APPLICATION_NAME = "spotify-game-app";
```

Comment:
Maybe change spotify to youtube to future proof this code, since we are going to be using youtube now.

Also while the order of the keywords final static private doesn't affect how the code works, generally its good to be consistent with a particular ordering.

https://stackoverflow.com/questions/2832126/quick-java-question-about-private-static-final-keywords-for-fields

Conventions you can read about in the answer to that question, but it is generally `private static final`

Reply:
We set the artifactId as "song-guessing-game", so I think it would make sense to use that:
https://github.com/googleinterns/step105-2020/pull/10/files